lesson2/task1_defining_variables/main.cpp

```diff
@@ -5,11 +5,18 @@
 using namespace std;
 
 int main() {
-    // TODO: Define variables
 
-    // TODO: Assign values
+    int a0 = 100;
+    char a1 = 'a';
+    bool Boolean = true;
```

Comment:
variables naming should start from small caharacter. Also the logic of naming is broken. Why not a2?